JavaScript/tests/syntax_test_typescript.ts

```diff
@@ -265,12 +265,23 @@ import foo;
 //                ^^^^ keyword.declaration.enum
 //                     ^^^ entity.name.enum
 
+    declare const foo: any;
+//  ^^^^^^^ storage.modifier
+//          ^^^^^ keyword.declaration
+//                ^^^ meta.binding.name variable.other.readwrite
+//                   ^ punctuation.separator.type
+//                     ^^^ meta.type support.type.any
+//                        ^ punctuation.terminator.statement
+
```

Comment:
Why is that not being added to the newly created `syntax_test_typescript_declarations.d.ts` where most other `declare` tests have been added?